### PR TITLE
Created type definitions for composer-concerto package

### PR DESCRIPTION
**Pull request by j4m3sb0mb**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/composer-concerto/index.d.ts
Comment (RyanCavanaugh) [inline types/composer-concerto/index.d.ts]: Typo

File: types/composer-concerto/index.d.ts
Comment (RyanCavanaugh) [inline types/composer-concerto/index.d.ts]: Check all declarations of `toString` for casing

**Review by RyanCavanaugh — CHANGES_REQUESTED** [review]

**Comment by j4m3sb0mb** [comment]
I've committed the requested changes, could you review it now?
thanks

**Review by j4m3sb0mb — COMMENTED** [review]
Changes are done, I'm waiting for another review

**Pull request closed without merging**
